.github/workflows/selfdrive_tests.yaml

```diff
@@ -157,6 +157,17 @@ jobs:
         $DOCKER_LOGIN
         docker push $DOCKER_REGISTRY/$CL_BASE_IMAGE:latest
 
+  validate_lock:
+    name: validate poetry lock
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: true
+    - uses: ./.github/workflows/setup
+    - name: Check poetry lock file
+      run: ${{ env.RUN }} "poetry lock --check"
```

Comment:
let’s just add this to the static analysis job

Reply:
added that to pre-commit hooks instead